### ISSUE TITLE
[Adaptive Cards - Rate your experience]: User is unable to activate “Your Privacy is important to us” link control while navigating using keyboard TAB key.

### ISSUE THREAD

**Issue by carlos-zamora** [microsoft/AdaptiveCards#7194]
https://microsoft.visualstudio.com/OS/_workitems/edit/38275651
